Sources/NIO/Selector.swift

```diff
@@ -248,7 +246,7 @@ final class Selector<R: Registration> {
     private typealias EventType = Epoll.epoll_event
     private let eventfd: Int32
     private let timerfd: Int32
-    private var earliestTimer: UInt64 = UInt64.max
+    private var earliestTimer: NIODeadline = .exactly(.max)
```

Comment:
we should have a constant for this called `.distantFuture` or `.forever` or something

Reply:
Yup, `.distantFuture` and `.distantPast` are the relevant prior art.